<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComparisonDialog</class>
 <widget class="QDialog" name="ComparisonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compare samples</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="aGroupBox">
        <property name="title">
         <string>Sample A</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="tabALabel">
           <property name="text">
            <string>Tab</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="tabAComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="entryALabel">
           <property name="text">
            <string>Entry</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="entryAComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="bGroupBox">
        <property name="title">
         <string>Sample B</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="tabBLabel">
           <property name="text">
            <string>Tab</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="tabBComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="entryBLabel">
           <property name="text">
            <string>Entry</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="entryBComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonsWidget" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="colorPushButton">
        <property name="text">
         <string>Mark color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acceptPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Compare</string>
        </property>
        <property name="icon">
         <iconset resource="gui_res.qrc">
          <normaloff>:/Images/icons/object-flip-horizontal.png</normaloff>:/Images/icons/object-flip-horizontal.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="gui_res.qrc"/>
 </resources>
 <connections/>
</ui>
